lesson3/MongoDB/README.md

```diff
@@ -40,7 +40,7 @@ ManogoDB Lesson 1 for Hack Your Future ***Class12***
 ## Installing MongoDB on Mac
 
 ##### First Install Homebrew
-`/usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"`
+`/usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/main/install)"`
```

Comment:
@martamatos can you double check this change?
 from https://raw.githubusercontent.com/Homebrew/install/main/install to https://raw.githubusercontent.com/Homebrew/install/master/install)